docs/snap-clone.md

```diff
@@ -42,7 +46,7 @@
 Also PVC cannot be deleted if it's having snapshots. Make sure all the snapshots
 on the PVC are deleted before you delete the PVC.**
 
-## Snapshot
+## Create CephFS snapshot and Clone Volume
```

Comment:
**snapshot** is owercase and **Volume** is uppercase. It would be nicer to have both **Snapshot** and **Volume**, here and below for RBD

Reply:
done